data/application.go

```diff
@@ -26,7 +26,7 @@ var (
 			URL:  "https://github.com/ibraginsky/book-server",
 		},
 	}
-	TackleTestapp = api.Application{
+	TackleTestappBin = api.Application{
```

Comment:
The application can be used not only for binary analysis.
I suggest using`TackleTestappPrivate`/`TackleTestappPublic`